### ISSUE TITLE
Comma-separated doxygen params: only the first one is highlighted

### ISSUE THREAD

**Issue by lamm45** [jeff-hykin/better-cpp-syntax#613]
In Doxygen, multiple parameters in a comma separated list will use the same description: https://doxygen.nl/manual/commands.html#cmdparam

However, currently only the first listed parameter is highlighted in a Doxygen comment block.

## Checklist

- [x] This problem exists even with the setting `"C_Cpp.enhancedColorization": "Disabled"`

## The code with a problem is:
```c++
/** Sets the position.
 *  @param x,y,z Coordinates of the position in 3D space.
 */
void setPosition(double x,double y,double z,double t)
{
}
```

## It looks like:
Default dark theme in VS Code 1.72.2:
![Screenshot_20221118_195658](https://user-images.githubusercontent.com/96844552/202826438-23391710-99f8-41e5-8cbe-82b225873a70.png)

## It should look like:
Variables x, y and z should all be in the same color in the comment block.

**Comment by jeff-hykin** [comment]
should now be fixed on `v1.17.2`

**Comment by sean-mcmanus** [comment]
This was fixed for C++, but it still repros with C, see https://github.com/microsoft/vscode-cpptools/issues/14104
